lib/std/os/bits/linux.zig

```diff
@@ -1448,3 +1449,7 @@ pub const MFD_HUGE_512MB = HUGETLB_FLAG_ENCODE_512MB;
 pub const MFD_HUGE_1GB = HUGETLB_FLAG_ENCODE_1GB;
 pub const MFD_HUGE_2GB = HUGETLB_FLAG_ENCODE_2GB;
 pub const MFD_HUGE_16GB = HUGETLB_FLAG_ENCODE_16GB;
+
+pub const NAME_MAX = 255;
```

Comment:
`NAME_MAX` belongs at the top next to e.g. `PATH_MAX`

Reply:
Should I also move `HOST_NAME_MAX`?

Reply:
> Should I also move `HOST_NAME_MAX`?

Actually `HOST_NAME_MAX` should be moved under `std/c`; it's not actually a kernel ABI.

The list of things that belong at the "top" of linux/bits is these: https://elixir.bootlin.com/linux/latest/source/include/uapi/linux/limits.h#L7 (excluding `NR_OPEN`)
```C
#define NGROUPS_MAX    65536	/* supplemental group IDs are available */
#define ARG_MAX       131072	/* # bytes of args + environ for exec() */
#define LINK_MAX         127	/* # links a file may have */
#define MAX_CANON        255	/* size of the canonical input queue */
#define MAX_INPUT        255	/* size of the type-ahead buffer */
#define NAME_MAX         255	/* # chars in a file name */
#define PATH_MAX        4096	/* # chars in a path name including nul */
#define PIPE_BUF        4096	/* # bytes in atomic write to a pipe */
#define XATTR_NAME_MAX   255	/* # chars in an extended attribute name */
#define XATTR_SIZE_MAX 65536	/* size of an extended attribute value (64k) */
#define XATTR_LIST_MAX 65536	/* size of extended attribute namelist (64k) */

#define RTSIG_MAX	  32
```